sofiadinizms/barbie-world
# feat(feature/home-sreen)

## What I did

1. Created the Home View
2. Added the elements shown on the Figma mockup for that screen
3. Adjusted the project settings from iOS to MacOS

## How to test

1. Clone this branch
2. Run the project
3. Check if the screen looks like the prototype (if needed, call me or Ryan to check if something is supposed to behave that way)
4. Go on the Project Info and check if the only targeted device is a Mac computer
